### PR TITLE
Registration page login styling

### PR DESCRIPTION
**Pull request by mendelB**
#### What's this PR do?
- changes the login link to a button and moves it to the top of the signup form for easier accessibility.

fixes https://www.pivotaltracker.com/story/show/151737409

#### How should this be reviewed?
large:
image

small:
image

medium:
image

#### Checklist
- [ ] Documentation added for changed endpoints.
- [ ] Tests added for new features/bug fixes.
- [ ] Post a message in #api if this includes something that causes a rebuild!  

---
For review: …

### CODE REVIEW

File: resources/assets/scss/_components/_std-login.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.std-login {
```
Comment (itsjoekent) [inline resources/assets/scss/_components/_std-login.scss]: not sure I'm a fan of the name, maybe just `_button.scss` and have a `.button.margin-md { ...`? im not the naming guru so i defer to dave/diego

Comment (DFurnes) [reply resources/assets/scss/_components/_std-login.scss]: Yeah, what is this class doing? If it's for a row of buttons, let's see if we can use the [Inline Form Actions](http://forge.dosomething.org/#forms-inline-form-action) pattern.

Comment (mendelB) [reply resources/assets/scss/_components/_std-login.scss]: Ah shucks. I kinda dig adding unnecessary classes and cluttering the CSS space but I'll give in here.

File: resources/views/auth/register.blade.php [hunk 2]
```diff
@@ -9,10 +9,18 @@
...
     <div class="container__block">
-        @include('auth.facebook')
+        <ul class="form-actions -inline">
+            <li>@include('auth.facebook')</li>
+            <li><a href="{{ url('login') }}" class="button">{{ trans('auth.log_in.default') }}</a></li>
+        </ul>
         <span class="divider"></span>
     </div>
 
+    <!-- <div class="contain">
```
Comment (DFurnes) [inline resources/views/auth/register.blade.php]: Can this commented markup be removed?

Comment (mendelB) [reply resources/views/auth/register.blade.php]: ach pardon me. yes!

**Pull request merged**